### ISSUE TITLE
missing header file

### ISSUE THREAD

**Issue by brancomat** [ARPA-SIMC/dballe#287]
Somewhere between v9.5 and v9.7 `/usr/include/dballe/msg/msg-extravars.h` has vanished from the installed files.

It's included in installed header `/usr/include/dballe/msg/msg.h`

it doesn't seem listed in the `install_headers` section of `dballe/msg/meson.build`

**Comment by brancomat** [comment]
Minor update: I tried adding `msg-extravars.h` but it resulted in a `dballe/msg/meson.build:41:0: ERROR: File msg-extravars.h does not exist.` (https://simc.arpae.it/moncic-ci/dballe/last/master/rocky8/build.log)

**Comment by edigiacomo** [comment]
I think I fixed the issue: the rpm built in a monci container contains the header

```
[root@mc-buyigo build]# rpm -qlp ~/rpmbuild/RPMS/x86_64/libdballe-devel-9.7-1.fc40.x86_64.rpm | grep msg-
/usr/include/dballe/msg/msg-extravars.h
```

**Comment by brancomat** [comment]
Fixed in dballe v9.9-1
